tests/base.py

```diff
@@ -0,0 +1,33 @@
+#
+# (c) Copyright 2019 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+import functools
+
+
+def addfixture(fixture, *fargs):
```

Comment:
I think this decorator is not that helpful. It reduces amount of typing in creating tests, but there is some functionality lost and in my opinion going more explicit it more useful for open source project. 

I would recommend to drop it and use already existing useFixture in Testcase and then use inheritance to reduce amount of boiler plate code.

usefixture allows to access fixture in setUp code which allows to use fixture in other fixture, something that could be useful.

Reply:
I've switched to `useFixture` but keeping them within the tests they impact.